source/js/main.js

```diff
@@ -111,7 +111,7 @@
                 toggleFold(this, fold === 'folded');
             });
 
-            $('figure.highlight figcaption .fold').click(function() {
+            $('figure.highlight figcaption').click(function() {
```

Comment:
Does this mean clicking the caption will always fold or unfold the code block even if folding is disabled in the `_config.yml`?

Reply:
No, this javascript is inside a `if (fold)` branch, so when folding is disabled in config, this line will not execute.